config/default.schema.json

```diff
@@ -8,12 +8,7 @@
 		},
 		"NODE_ENV": {
 			"type": "string",
-			"enum": [
-				"development",
-				"test",
-				"production",
-				"migration"
-			],
+			"enum": ["development", "test", "production", "migration"],
```

Comment:
The formatter formatted a lot of lines in this file. We should undo the formatting changes.